tests/setupTestsAfterEnv.ts

```diff
@@ -0,0 +1,16 @@
+import path from 'path';
+import { setupPolly } from 'setup-polly-jest';
+import NodeHttpAdapter from '@pollyjs/adapter-node-http';
+import FSPersister from '@pollyjs/persister-fs';
+
+global.pollyContext = setupPolly({
+  adapters: [NodeHttpAdapter],
+  persister: FSPersister,
+  persisterOptions: {
+    fs: {
+      recordingsDir: path.resolve(__dirname, '__recordings__'),
+    },
+  },
+  recordIfMissing: false,
```

Comment:
Usually Polly records requests by default. That is fine if you are adding Polly to a fresh project, but since we're adding Polly to an existing project we need to know whether there are requests being made in tests that we forgot to mock using Nock. This allows us to do this — we will see a warning if Polly sees a request it doesn't recognize.

Reply:
Hmm. I was expecting there to be some way to throw if a record is missing. Otherwise we might miss these warnings creeping in, and we might not see tests fail either for the same reason you mentioned for `disableNetConnect`.

Reply:
Honestly this is the main sticking point for me at the moment. I'd feel better knowing that it was impossible for it to hit the network. Is it possible for us to add a catch-all mock that throws an error, something that would act similarly to nock's `disableNetConnect`?

Reply:
I misspoke: it *does* throw if a recording is missing. This assumes that nothing will catch that error, though. CollectiblesController has a fair amount of `try`/`catch`s, so what would have been errors were simply warnings.

Reply:
Oh... that's great! Well. I am for switching to Polly then. Seems equally effective and at least a bit easier to use than nock.